### DISCUSSION TITLE
Option to keep the bigger, the newer or a combination of both

### DISCUSSION THREAD

**Discussion by liam-maps** [qarmin/czkawka#495] [Ideas]
Hi.
I'm using your software more frequently every day and I want to thank you for creating it.
I'm trying to merge some folders with mixed file types (binaries, pictures, videos...) and some times I'd like to keep the bigger of both duplicates (searching by name) because I'm sure the bigger has more contents in it and that's the one I'd like to keep.

Also, sometimes I'd be great to check if newer, and then check if bigger, because sometimes something goes wrong, and the older keeps more info in it, and thats the one I'd like to keep, not because it's older, but because is bigger.

Thanks again for your work.
Cheers.

**Comment by x-ray-romanoff** [comment]
@pr-Fi-- -

**Comment by awesome14** [comment]
Hi Liam-maps, it sounds like your looking a duplicate _file name_ finder, which is different than a duplicate  _file_ finder. It's tough to imagine adding something to a file and having the larger file be older than the smaller one. Rsync does pretty much what you want, but I'm not sure about pruning off the paths, i.e., putting everything together in one directory.  You could take a look at bleachbit, also. Or, maybe the developer will add what you want.
